### PR TITLE
created the needed settings to import the project successfully into p…

### PR DESCRIPTION
**Pull request by lhein**
…hoton

Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [ ] Did you use the Jira Issue number in the commit comments?
- [ ] Did you set meaningful commit comments on each commit?
- [ ] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

File: core/plugins/org.fusesource.ide.tooling/.classpath [hunk 1]
```diff
@@ -1,6 +1,11 @@
...
 <classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">
+		<attributes>
+			<attribute name="maven.pomderived" value="true"/>
+		</attributes>
+	</classpathentry>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
-	<classpathentry kind="output" path="bin"/>
+	<classpathentry kind="src" path="src/"/>
```
Comment (apupier) [inline core/plugins/org.fusesource.ide.tooling/.classpath]: this folder doesn't exist and report an error for me

**Comment by apupier** [comment]
I have several errors like that:
"The project was not built due to "Resource already exists on disk: '/org.fusesource.ide.camel.model.service.impl/target/classes/org/fusesource/ide/camel/model/service/internal/camelContext-2.10.x.json'.". Fix the problem, then try refreshing this project and building it since it may be inconsistent"

I need to refresh the target folder to get rid of it

File: core/plugins/org.fusesource.ide.tooling/.classpath (hunk 1 above)
Comment (lhein) [reply core/plugins/org.fusesource.ide.tooling/.classpath]: odd...for me it doesn't

**Comment by lhein** [comment]
not required

**Pull request closed without merging**